modules/migration-compatibility-guidelines.adoc

```diff
@@ -20,29 +20,32 @@ modern operator:: The {mtc-short} Operator designed for modern platforms.
 control cluster:: The cluster that runs the {mtc-short} controller and GUI.
 remote cluster:: A source or destination cluster for a migration that runs Velero. The Control Cluster communicates with Remote clusters via the Velero API to drive migrations.
 
-
-[cols="1,2,2", options="header"]
 .{mtc-short} compatibility: Migrating from a legacy platform
 |===
-||{product-title} 4.5 or earlier |{product-title} 4.6 or later
-|Stable {mtc-short} version a|{mtc-short} {mtc-version}._z_
+| |{product-title} 3.11 |{product-title} 4.0 to 4.5 |{product-title} 4.6 or later
+
+|Stable {mtc-short} version
+|{mtc-short} v.1.7._z_
+|{mtc-short} v.1.7._z_
+|{mtc-short} v.1.7._z_
+
+| N/A
+| N/A
+|Legacy {mtc-short} v.1.7._z_ operator: Install manually with the `operator.yml` file.
 
-Legacy {mtc-version} operator: Install manually with the `operator.yml` file.
 [IMPORTANT]
-====
+
 This cluster cannot be the control cluster.
-====
 
-|{mtc-short} {mtc-version}._z_
 
-Install with OLM, release channel `release-v1.7`
+|Install with OLM, release channel `release-v1.7`
 |===
 
 [NOTE]
 ====
 Edge cases exist in which network restrictions prevent modern clusters from connecting to other clusters involved in the migration. For example, when migrating from an {product-title} 3.11 cluster on premises to a modern {product-title} cluster in the cloud, where the modern cluster cannot connect to the {product-title} 3.11 cluster.
```

Comment:
Although not part of your content, would converting this note to normal text work better? The note takes up more word-real-estate that the preceeding text.

Reply:
Good idea. Thank you!